api/src/main/java/pl/kolendateam/dadcard/abilitys/MapperAbilitysToDTO.java

```diff
@@ -0,0 +1,26 @@
+package pl.kolendateam.dadcard.abilitys;
+
+import pl.kolendateam.dadcard.abilitys.dto.AbilitysDTO;
+import pl.kolendateam.dadcard.abilitys.entity.Abilitys;
+
+public class MapperAbilitysToDTO {
+    public static AbilitysDTO toAbilityDTO(Abilitys abilitys){
+        
+        AbilitysDTO abilitysDTO = new AbilitysDTO(
+            abilitys.getStreght(),
```

Comment:
Myślę że wystarczy zwracać na widok wartość siły a bonus już sobie widok policzy bo wzór jest prosty

Reply:
to nie takie łatwe :) wydaje mi się że trzeba rozdzielić wartość od bonusu tej wartości. Tu jak wyglądać będzie kompletna tabelka samych bonusów. Tam wpadną bonusowe zaklęcia i możliwość rzucania ich od pewnego poziomu wartości (więcej niż 10).
https://www.d20srd.org/srd/theBasics.htm#abilityScores